set.c

```diff
@@ -2132,7 +2194,13 @@ Init_Set(void)
     rb_define_method(rb_cSet, "superset?", set_i_superset, 1);
     rb_define_alias(rb_cSet, ">=", "superset?");
     rb_define_method(rb_cSet, "to_a", set_i_to_a, 0);
+    rb_define_method(rb_cSet, "to_h", set_i_to_h, 0);
     rb_define_method(rb_cSet, "to_set", set_i_to_set, -1);
+    // rb_define_singleton_method(rb_cSet, "from_hash", compat_loader, 1);
```

Comment:
Is there a purpose to this commented out method?

Reply:
I assume that leaving it commented out makes it easier to test manually.  However, it seems like it should be a regular method and not a singleton method, as `compat_loader` calls `set_i_to_hash`, which accepts a Set instance and not a class. @nobu is that correct?

Reply:
Yes, it was for just debugging but a mistake.
Please remove it.

Reply:
Pushed a commit to remove it.  I'll be squashing these commits when merging.